app/views/comments/_comment.html.erb

```diff
@@ -30,7 +30,7 @@
     </div>
 
     <div class="content">
-      <%= simple_format(comment.content) %>
+      <%= simple_format(h(comment.content)) %>
```

Comment:
according to [the docs](url
https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format), `simple_format` already does this.